app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,41 @@
+class VotesController < ApplicationController
+
+  def index
+    @parent = Question.find(params[:question_id]) if params[:question_id]
+    @parent = Answer.find(params[:answer_id]) if params[:answer_id]
+    @votes = Vote.all
```

Comment:
What was the intention here - it looks like you are loading a question into @parent then immediately overwriting it with an answer